lib/smsHandler.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+var config = require('../config'),
+    constant = require('../constants'),
+    responses = require('./buildResponses'),
+    logger = require('logops');
+
+function processMessage(req, res) {
+//function that checks the received message: ABCDGC|ABCDGP|ABDCMC|ABCDMP
+
+    var receivedMessage = req.body.message.substring(4, 6);
+    logger.info('Received message: ' + receivedMessage);
+
+    // ABCDGC
+    if (receivedMessage = constant.GC_MESSAGE) {
+        res.status(constant.HTTP_RESPONSE_OK).send('OK');
+        responses.createRCMessage();
+    }
+    // ABCDGP
+    else if (receivedMessage = constant.GP_MESSAGE) {
+        res.status(constant.HTTP_RESPONSE_OK).send('OK');
+        responses.createRPMessage();
+    }
+    // ABCDMC
+    else if (receivedMessage = constant.MC_MESSAGE) {
+        res.status(constant.HTTP_RESPONSE_OK).send('OK');
+        responses.createContextBrokerNotification();
+    }
+    // ABCDMP
+    else if (receivedMessage = constant.MP_MESSAGE) {
+        res.status(constant.HTTP_RESPONSE_OK).send('OK');
+        responses.createContextBrokerNotification();
```

Comment:
so... it's called responses, it will make a creation... but in fact it's going to send a request... ¿aposta?

Reply:
It smells like a refactor here and in buildResponses. What if we just build the payloads at buildResponses, and we make the request here (with invokeDashboard(responseString), Option object doesn't seem necessary as a param).